_posts/2015-02-19-Implicit_Mentorship.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: The importance of Implicit Mentorship
+category : Mentorship
+tagline: "What makes Implicit Mentorship different?"
+author: Steven McDonald
+tags: [Mentorship]
+---
+{% include JB/setup %}
+
+I think everyone understands that having an explicit mentor is a great thing that has many benefits but I sometimes I don't think that people think of Implicit Mentorship.
+
+Explicit Mentorship is an intentional process and is generally a relationship that is agreed on between two or more people. You can discuss topics with your mentor and get feedback and insights on various topics.
+
+##Implicit Mentorship?
+
+In the same way that a natural leader can emerge from certain situations I think mentors can also emerge. Often these people do not intend to mentor or even realize that they are mentoring someone else.
+
+Why is Implicit Mentorship important though? On a number of occassions I have seen situations where someone could greatly benefit from somebody else's experience. When making the suggestion that person X could learn from person Y, either person Y does not feel comfortable "teaching" or person X dismisses that they would gain anything of significance. However, if these people stumble onto the topic themselves, a conversation usually takes place and possibly even a transfer of knowledge no matter how small.
+
+In my opinion this is why developers pairing on a problem can be so valuable. Not only has the problem at hand been addressed by two developers covering each other, there is also a transfer of knowledge that takes place that has nothing to do with the actual problem at hand. When one developer shows another developer a new technique, a new editor, a new shortcut or anything of the sort they are mentoring one another.
```

Comment:
I don't fully understand what you mean by covering each other. This could either be left out or expanded on further.

You last point about any learning being mentoring - maybe establish upfront what Mentorship is before making the leap that this is mentoring